### PR TITLE
[slider] add haptic feedback when slider changes page

### PR DESCRIPTION
**Pull request by annuhdo**
# Context
Issue: https://github.com/Aidoku/Aidoku/issues/191

- Added selection haptic feedback when `ReaderViewController#displayPage` gets called.
- Stored `currentPageLabelVal` to `ReaderToolbarView.swift` so we can figure out if the user had slid to a page that differed from previous page. We only fire the haptic feedback when the page changes.

# Test
- Can't build this on physical device so I can't really test.
- Ran print statements which ran in the if block.

----

I'm new to iOS development so would appreciate any feedback! I'm also aware that the test plan is lacking so feel free to reject the PR but pls provide feedback on code if possible :)

### CODE REVIEW

**Comment by ItsLame** [comment]
Just tested it on my iPhone. Rather than the haptic feedback kicking in when page changes, it instead continuously triggers haptic when dragging the slider.

**Comment by annuhdo** [comment]
@ItsLame oof. Let me try again and test on a test project.

Not familiar with how to set this PR as planning changes but uh —planning changes— now.

**Comment by annuhdo** [comment]
Will put up another PR

**Comment by ItsLame** [comment]
@annuhdo I think you can just make updates to your current branch (`hapticslider`) and it'll show up here

**Comment by ItsLame** [comment]
> Not familiar with how to set this PR as planning changes but uh —planning changes— now.

Also for this I usually just set the PR back to draft like so:
image

**Comment by ItsLame** [comment]
@annuhdo just noticed, I think you forgot to re-open this PR?

File: iOS/UI/Reader/ReaderToolbarView.swift [hunk 1]
```diff
@@ -8,7 +8,19 @@
...
+            if previousPageValue != currentPageValue {
+                previousPageValue = currentPageValue
+            }
+        }
+    }
+    var previousPageValue: Int? {
+        willSet {
+            let feedbackGenerator = UISelectionFeedbackGenerator()
+            feedbackGenerator.selectionChanged()
+        }
```
Comment (Skittyblock) [inline iOS/UI/Reader/ReaderToolbarView.swift:22]: you don't need two variables, just use `oldValue`
```suggestion
            if oldValue != currentPageValue {
                let feedbackGenerator = UISelectionFeedbackGenerator()
                feedbackGenerator.selectionChanged()
            }
        }
```

**Review by Skittyblock — CHANGES_REQUESTED** [review]
